tests/test_helpers.py

```diff
@@ -190,5 +194,41 @@ def test_extract_product_name(self):
 
         self.assertEqual(short_name_want, short_product_name)
 
+
+    # Filenames to test markdown syntax highlight with.
+    test_markdown_filenames = [
+        [
+            "tests/markdown_syntax_highlight.md",
+            "tests/markdown_syntax_highlight_got.md",
```

Comment:
We should either use a tmp file, clean these up every time (not after the assert), or include them in `.gitignore`.